compaction/compaction.cc

```diff
@@ -753,6 +755,7 @@ class compaction {
             // for a better estimate for the number of partitions in the merged
             // sstable than just adding up the lengths of individual sstables.
             _estimated_partitions += sst->get_estimated_key_count();
+            _estimated_droppable_tombstone_ratio += sst->estimate_droppable_tombstone_ratio(gc_clock::now(), _table_s.get_tombstone_gc_state(), _schema);
             _compacting_data_file_size += sst->ondisk_data_size();
```

Comment:
@lkshminarayanan I remember droppable ratio can go above 1.0.

the calculation is (droppable tombstone count ) / (cell count)

perhaps there are certain tombstone types that aren't accounted as a cell, maybe for non-atomic types (e.g. collections). https://github.com/scylladb/scylla-enterprise/issues/1624

let's account for that, just to be on safe side, by getting minimum of 1.0f and estimate_droppable_tombstone_ratio()

Reply:
Done.